### ISSUE TITLE
Enable alerts to only go to contacts/contactgroups on service definition

### ISSUE THREAD

**Issue by tmcnag** [NagiosEnterprises/nagioscore#261]
> This issue has been automatically migrated from [http://tracker.nagios.org/view.php?id=0000535](http://tracker.nagios.org/view.php?id=0000535)  - Please follow that link for details.
> Originally submitted on 2013-11-25 by BanditBBS

 In most cases, sending notifications to the contacts/contactgroups on the host and the ones defined on the service makes sense. But what if the service doesn't pertain to the others. Need some method to only use the contacts/contactgroups that are defined on the service definition and to ignore those on the host.

**Comment by jfrickson** [comment]
This will probably be something we can do in Core 5.

**Comment by jfrickson** [comment]
I set up a test where there were no contacts or contact_groups on a service or any of the templates. Then I set `test1` to be the contract_group on the host. After restarting, `test1` showed up in contact_groups for the service, as expected. I then set `test2` to be the contract_group on the service. After restarting, only `test2` showed up in contact_groups for the service. So it appears this is no longer a problem.
